ConfigurationSystem/private/Refresher.py

```diff
@@ -143,6 +143,11 @@ def __refresh( self ):
     else:
       lInitialListOfServers = gConfigurationData.getServers()
       gLogger.debug( "Refreshing from list %s" % str( lInitialListOfServers ) )
+
+    # If no servers in the initial list, we are supposed to use the local configuration only
+    if not lInitialListOfServers:
+      return S_OK()
+
```

Comment:
This is a copy paste of the fix you did in v6r13 : https://github.com/DIRACGrid/DIRAC/commit/fbb3b5f187e42e2a3473eab056634df562bff295